LibertyDSNP/test-generators
# udpate our exports so package users can get to all of them

Problem
=======
I wasn't correctly exporting the things we need to use this package; in sdk I couldn't get at anything in sampleText or addresses.  This should fix it.

Change summary:
---------------
* import the files into index.ts and export those things as module consts
* Also make the tool-versions file match what's in the sdk so npm link works, too

Steps to Verify:
----------------
You should be able to `npm install` this package and use all functions from index, sampleText AND addresses.
